bedrock/pocket/templates/pocket/add.html

```diff
@@ -26,84 +26,84 @@
+               <li class="app-store-badge" aria-label="{{ ftl('pocket-mac-app-store') }}"><a href="https://apps.apple.com/app/pocket/id568494494?ls=1&mt=12" target="_blank" rel="noopener noreferrer" class="dark-inline-link"><img src="{{ static('img/pocket/app-button-mac.png') }}" alt=""></a></li>
+               <li class="app-store-badge" aria-label="{{ ftl('pocket-google-play-store') }}"><a href="https://play.google.com/store/apps/details?id=com.ideashower.readitlater.pro" target="_blank" rel="noopener noreferrer" class="dark-inline-link"><img src="{{ static('img/pocket/app-button-google.png') }}" alt=""></a></li>
+               <li class="app-store-badge" aria-label="{{ ftl('pocket-amazon-app-store') }}"><a href="https://www.amazon.com/dp/B0057PAY8G" target="_blank" rel="noopener noreferrer" class="dark-inline-link"><img src="{{ static('img/pocket/app-button-amazon.png') }}" alt=""></a></li>
             </ul>
             <ul class="device-list">
-               <li><a href="https://chrome.google.com/webstore/detail/save-to-pocket/niloccemoadcdkdjlinkgdfekeahmflj?hl=en" target="_blank" rel="noopener noreferrer" class="dark-inline-link rarrow">Windows</a></li>
-               <li><a href="https://help.getpocket.com/article/1006-getting-started-with-kobo" target="_blank" rel="noopener noreferrer" class="dark-inline-link rarrow">Kobo e-reader</a></li>
-               <li><a href="https://getpocket.com/apps/windowsphone/" target="_blank" rel="noopener noreferrer" class="dark-inline-link rarrow">Windows Mobile</a></li>
-               <li><a href="https://getpocket.com/apps/blackberry/" target="_blank" rel="noopener noreferrer" class="dark-inline-link rarrow">Blackberry</a></li>
-               <li><a href="https://getpocket.com/apps/" target="_blank" rel="noopener noreferrer" class="dark-inline-link rarrow">More Apps</a></li>
+               <li><a href="https://chrome.google.com/webstore/detail/save-to-pocket/niloccemoadcdkdjlinkgdfekeahmflj?hl=en" target="_blank" rel="noopener noreferrer" class="dark-inline-link rarrow">{{ ftl('pocket-windows') }}</a></li>
+               <li><a href="https://help.getpocket.com/article/1006-getting-started-with-kobo" target="_blank" rel="noopener noreferrer" class="dark-inline-link rarrow">{{ ftl('pocket-kobo-e-reader') }}</a></li>
+               <li><a href="https://getpocket.com/apps/windowsphone/" target="_blank" rel="noopener noreferrer" class="dark-inline-link rarrow">{{ ftl('pocket-windows-mobile') }}</a></li>
+               <li><a href="https://getpocket.com/apps/blackberry/" target="_blank" rel="noopener noreferrer" class="dark-inline-link rarrow">{{ ftl('pocket-blackberry') }}</a></li>
+               <li><a href="https://getpocket.com/apps/" target="_blank" rel="noopener noreferrer" class="dark-inline-link rarrow">{{ ftl('pocket-more-apps') }}</a></li>
```

Comment:
Nit: remove `target="_blank" rel="noopener noreferrer"` on getpocket.com links

Reply:
Should this also be the case for `help.getpocket.com` links?